core/modules/system/system.pages.inc

```diff
@@ -8,35 +8,264 @@
  * Page callback to output a token tree as an empty page.
  */
 function system_token_output() {
-  $options = isset($_GET['options']) ? backdrop_json_decode($_GET['options']) : array();
+  $types = isset($_GET['token_types']) ? backdrop_json_decode($_GET['token_types']) : array();
+  $globals = isset($_GET['global_types']) ? backdrop_json_decode($_GET['global_types']) : TRUE;
+  $query = backdrop_get_query_parameters();
+  $show_restricted = $query['show_restricted'];
 
-  // Check the token against the serialized options to prevent random access to
-  // the token browser page.
-  if (!isset($_GET['token']) || !backdrop_valid_token($_GET['token'], 'token-tree:' . serialize($options))) {
-    return MENU_ACCESS_DENIED;
-  }
-
-  $tree = '<div>' . theme('token_tree', $options) . '</div>';
+  $out = system_token_browser_tree_build_content($types, $globals, $show_restricted);
+  $out = backdrop_render($out);
 
   if (backdrop_is_ajax()) {
-    // Return as a renderable so the Backdrop dialog system can use the title.
     $dialog_options = array(
-      'dialogClass' => 'token-dialog',
+      'dialogClass' => 'token-browser-dialog',
+      'width' => 900,
```

Comment:
This width => 900 gets overridden by width => '800' just a few lines later.

Reply:
Done.